cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala

```diff
@@ -1,5 +1,5 @@
 /***
- *   Copyright 2014 Rackspace US, Inc.
+ *   Copyright 2015 Rackspace US, Inc.
```

Comment:
Don't change the date:

http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated